docs/wallets.md

```diff
@@ -28,18 +28,18 @@ codex32 shares may be any length between 128 and 512 bits.
 Wallets should support import of 128- or 256-bit seeds; other lengths are optional.
 
 128-bit seeds are 48 characters in length, including the `MS1` prefix.
-256-bit seeds are 74. For other bitlengths, see the BIP.
+256-bit seeds are 74. For other bit-lengths, see the BIP.
 
 The process for entering shares is:
 
-1. The user should select the bit length of the import before entering any actual data.
-1. Then the user should enter the first share. To the entext possible given screen limitations, when entering share data, data should be displayed in uppercase, visually separated into four-character windows. The first four-character window should include the `MS1` prefix, which should be pre-filled.
+1. The user should enter the first share. To the extent possible given screen limitations, when entering share data, data should be displayed in uppercase, with visually distinct four-character windows. The first four-character window should include the `MS1` prefix, which should be pre-filled.
 1. Once the first share is fully entered, the wallet should validate the checksum and header before accepting it.
-   * The user should not be able to entire mixed-case or non-bech32 characters.
+   * The user should not be able to entire mixed-case characters. The user must be able to enter all bech32 characters as well as `?` indicating an erasure. Wallets may allow users to enter non-bech32 characters, at their discretion. (This may be useful to guide error correction, e.g. by attempting to replace all `B`s with `8`s.)
```

Comment:
should not be able to entire

should not be able to enter

Reply:
The user should not be able to enter a previously used share index at position 9. If they do the user should be immediately warned to enter a different codex32 share than previously entered. "I am" converts it to an erasure ?, while "Thanks/OK" leaves the character unfilled and lets them start over.